go/tosca/interpreter.go

```diff
@@ -190,6 +190,7 @@ const (
 	R11_Paris
 	R12_Shanghai
 	R13_Cancun
+	R99_UnknownNextRevision
```

Comment:
Since the `R99_UnknownNextRevision` is not a real revision, I think it should be defined in `go/ct/common/revisions.go` together with other constants like `Min` and `Max` revision. It is OK if this leads to a print that looks like `Revision(99)` instead of a `UnknownNextRevision`.